docs/dev/coding_standards.rst

```diff
@@ -19,9 +19,13 @@ Many editors/IDEs can run this tool in the background while you work, showing re
 
 .. _flake8: https://flake8.pycqa.org/en/latest/
 
-It is allowed but not required to use the `black`_ code formatter. Do only format files that you are actively working on and make a dedicated commit for the initial formatting when working on an existing file that does not adhere to the black code style to facilitate code review. If there are conflicts between black's output and flake8 (especially related to `E203`_), flake8 takes precedence. Use ``#noqa : E203`` to disable E203 warnings for a specific line if appropriate.
+It is allowed but not required to use the `black`_ code formatter. 
+To avoid introducing unrelated changes when working on an existing file, consider using the `darker`_ tool.
```

Comment:
Should I formulate stronger here? "use the `darker` tool"?

Reply:
You can formulate it stronger, but not as strong: "it is recommended".

I'd add the reason as well: "to focus the changes on the implementation instead of unrelated formatting, helps the maintainers reviewing the code."
(it's not yet perfect, but I hope you get the idea)

Reply:
Sounds good, thanks!